src/client/components/admin/UserContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import Users from './Users';
+import UserTableRow from './UserTableRow';
+
+class UserContainer extends React.Component< {}, {
```

Comment:
I'm not a fan of calling this a Container because it's technically not a redux container